src/main/java/edu/ucsb/cs156/example/documents/RedditT3.java

```diff
@@ -0,0 +1,15 @@
+package edu.ucsb.cs156.example.documents;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class RedditT3 {
```

Comment:
Just curious, what does T3 refer to?

Reply:
I have no idea.  It's a Reddit thing.   I was trying to crank this out, and so I didn't dive into the Reddit documentation too much; I just sort of took everything at face value.